Plugins/GeoModel/include/Acts/Plugins/GeoModel/detail/GeoSubtractionConverter.hpp

```diff
@@ -0,0 +1,35 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Plugins/GeoModel/GeoModelDetectorElement.hpp"
+#include "Acts/Utilities/Result.hpp"
+
+#include <memory>
+#include <tuple>
+
+#include <GeoModelKernel/GeoShapeSubtraction.h>
+
+namespace Acts::detail {
+struct GeoSubtractionConverter {
+  /// @brief Convert a GeoBox to a detector element and surface
+  ///
+  /// @param geoFPV The full physical volume to convert (contains shape)
+  /// @param geoBox The GeoBox to convert
+  /// @param absTransform from the GeoPhysVol
```

Comment:
geoBox and then geoPolygon (in the paramaters)?

Reply:
I didn't know that this mistake could compile :sweat_smile: